cmd/estat

```diff
@@ -25,9 +25,8 @@ from bcc import BPF
 import getopt
 from glob import glob
 import os
-import subprocess
 import sys
-from time import sleep, time, strftime
+from time import sleep, strftime
 
 #
 # Find BCCHelper. If we are being run from the repo, we should be able to find
```

Comment:
This comment looks incomplete, can you fix that while you're here?

Reply:
This looks complete to me. Can you say what's missing? Or maybe it's worded a bit awkwardly?

Reply:
There's no subject in that last sentence, it's not a syntactically valid English sentence.

Reply:
I don't see the issue, but maybe I'm just a bit slow today. Maybe removing use of the imperative mood would be helpful?
> If we can't find that directory, we should look for BCCHelper in its install location.

Reply:
Oh, I see what the comment means to say now that you reword it that way. Based on the wording, I thought it was implying that if we aren't running from the repo, then we wouldn't be able to find (whatever it is we're finding).

Reply:
How about? 
# We need to find BCCHelper.  If being run from the repo, we should find
 # it in the repo's lib/ directory.  If we can't find that, look for BCCHelper
 # in its install location.

Reply:
I fixed the flake8 errors that came in during the rebase.

Reply:
Sounds great!

Reply:
nit: 'If we can't find that,' might read a bit better as 'If we can't find that _directory_,'. Other than that, sounds good.